include/js.h

```diff
@@ -1221,6 +1221,31 @@ js_delete_element(js_env_t *env, js_value_t *object, uint32_t index, bool *resul
   return js_convert_from_status(status);
 }
 
+static inline int
+js_set_array_elements(js_env_t *env, js_value_t *array, const js_value_t *elements[], size_t len, size_t offset) {
+  napi_status status = 0;
+  for (size_t i = 0; i < len; i++) {
+    status = napi_set_element(env, array, offset + i, (js_value_t *) elements[i]);
+    if (status != 0) break;
+  }
+  return js_convert_from_status(status);
+}
+
+static inline int
+js_get_array_elements(js_env_t *env, js_value_t *array, js_value_t **elements, size_t len, size_t offset, uint32_t *result) {
+  napi_status status = 0;
+  *result = 0;
```

Comment:
`result` is an optional pointer so there's no guarantee that it's set. We should do the same as here: https://github.com/holepunchto/libjs/blob/de90f0a72868fbd0d57850f7ec517f011fc979fd/src/js.cc#L5787-L5807